### ISSUE TITLE
+New Project : Text-aligment is not consistent 

### ISSUE THREAD

**Issue by coderhacker** [appirio-tech/connect-app#3898]
### Expected behavior
Text alignment should be consistent for below item:

a. Where to Share Style & Branding Guidelines.
b. Where to Share Font Preferences
c. Where to Share Color/Theme Preferences

### Actual behavior
ext alignment in-consistent for below item:

a. Where to Share Style & Branding Guidelines.
b. Where to Share Font Preferences
c. Where to Share Color/Theme Preferences

### Steps to reproduce the problem
Step 1-  go to https://connect.topcoder-dev.com and login 
Step 2.- login with username: topcoderconnect+AccountManager/appirio123.
Step 3.-  https://connect.topcoder-dev.com/new-project/app_new_intake see attached screenshot for more details.

### Screenshot/screencast
Attach or link a resource.

![image](https://user-images.githubusercontent.com/62358241/79305800-af461a00-7f11-11ea-92d6-7662cedad440.png)
![image](https://user-images.githubusercontent.com/62358241/79305836-bc630900-7f11-11ea-97a4-e0ee200f6035.png)

--

#### Environment
- OS:  Window 10
- Browser (w/version): Firefox (latest version)
- User role (client, copilot or manager): manager, 
- Account used: topcoderconnect+AccountManager

**Comment by vikasrohit** [comment]
I don't think it is possible to align them without increasing the space between word i.e. without justified alignment of text.

**Comment by coderhacker** [comment]
Just I have done some CSS trick on the browser and seems to look fine with the previous one.
![image](https://user-images.githubusercontent.com/62358241/79317246-1cae7680-7f23-11ea-8483-c7c92d0f84dc.png)
